engine/flink/kafka-util/src/main/scala/pl/touk/nussknacker/engine/kafka/generic/sinks.scala

```diff
@@ -16,14 +16,10 @@ object sinks {
     extends KafkaSinkFactory(GenericJsonSerialization, processObjectDependencies)
 
   case class GenericJsonSerialization(topic: String) extends SimpleSerializationSchema[Any](topic, element => {
-      val objToEncode = element match {
-        // TODO: would be safer if will be added expected type in Sink and during expression evaluation,
-        // would be performed conversion to it
-        case TypedMap(fields) => fields
-        case other => other
-      }
-      encoder.encode(objToEncode).spaces2
+    // TODO: would be safer if will be added expected type in Sink and during expression evaluation,
```

Comment:
Is this comment still relevant?

Reply:
I think so - this sink still accept anything, nothing changed in this place